lib/main.js

```diff
@@ -7,6 +8,10 @@ var showHotKey = Hotkey({
   }
 });
 
+innerHTML = 'foo!';
```

Comment:
A compile-time error in the JavaScript halted validation of that file.Message: missing ; before statement